tests/rados/mute_alerts.py

```diff
@@ -0,0 +1,238 @@
+    """
+    Mutes the health alert generated on the cluster
+    Args:
+        alert: Name of the alert to be muted
+        node: node on which command should be executed
+        duration: duration for which the alert should be muted.
+                Allowed Values: None -> mutes the specified alert indefinitely until the same alert is raised again
+                                5m, 1h -> mutes the specified alert for specified duration
+        sticky: makes use of the "--sticky" param to mute specified alert indefinitely
+
+    Returns: 0 -> pass, 1 -> failure
+
+    """
+    log.debug("Collecting all the alerts generated on the cluster")
+    all_alerts = get_alerts(node)
+    if alert not in all_alerts["active_alerts"] + all_alerts["muted_alerts"]:
+        log.info(f"the alert: {alert} not generated on the cluster, Cannot mute")
+        return 0
+    if alert in all_alerts["muted_alerts"]:
+        log.info(f"the alert: {alert} is already muted")
+        return 0
+
+    # Muting the given alert along with specified duration and
+    cmd = f"ceph health mute {alert} {duration if duration is not None else ''} {'--sticky' if sticky else ''}"
+    node.shell([cmd])
+    time.sleep(1)
+    all_alerts = get_alerts(node)
+    log.info(
+        f"Muted the alert : {alert}. All the muted alerts : {all_alerts['muted_alerts']}"
+    )
+    return 0 if alert in all_alerts["muted_alerts"] else 1
+
+
+def unmute_health_alert(alert: str, node: CephAdmin) -> int:
+    """
+    Un-Mutes the health alert on the cluster
+    Args:
+        alert: Name of the alert to be muted
+        node: node on which command should be executed
+
+    Returns: 0 -> pass, 1 -> failure
+
+    """
+    log.debug("Collecting all the alerts generated on the cluster")
+    all_alerts = get_alerts(node)
+    if alert not in all_alerts["muted_alerts"] + all_alerts["active_alerts"]:
+        log.info(f"the alert: {alert} not generated on the cluster, Cannot mute")
+        return 0
+    if alert in all_alerts["active_alerts"]:
+        log.info(f"the alert: {alert} is already un-muted")
+        return 0
+
+    # Un-Muting the given alert
+    cmd = f"ceph health unmute {alert}"
+    node.shell([cmd])
+    time.sleep(1)
+    all_alerts = get_alerts(node)
+    log.info(
+        f"Un-Muted the alert : {alert}. All the Un-muted alerts : {all_alerts['active_alerts']}"
+    )
+    return 0 if alert in all_alerts["active_alerts"] else 1
+
+
+def generate_health_alert(
+    alert: str, node: CephAdmin, clear: bool = False, **kwargs
+) -> int:
+    """
+    Method to generate various health alerts
+    Args:
+        alert: name of the alert to be generated
+        node: name of the installer node
+        clear: Bool value which specifies if the given alert should be cleared
+        kwargs: any other params that need to be sent for a particular alert
+
+    Returns: 0 -> pass, 1 -> failure
+
+    """
+    if alert == "OSDMAP_FLAGS":
+        flag = kwargs.get("flag")
```

Comment:
Looks like `flag` is mandatory. If yes, let's raise an attribute error by accessing the value using `kwargs["flag"]` instead of the command failing at `ceph osd set `.